doc/docs/development/system-tests.md

```diff
@@ -131,8 +131,6 @@ Use default executable directory: /workspaces/ankaios/tools/../target/debug
 Found ank 0.1.0
 Found ank-server 0.1.0
 Found ank-agent 0.1.0
-podman service not running -> start podman service
```

Comment:
The description above says:

> 3. It starts the `podman` service if it is not started yet.

Is it still true?

Reply:
Removed.